content/v0.1.markdown

```diff
@@ -0,0 +1,122 @@
+Title: Version 0.1 of Smithay
+Date: 2017-10-05 17:00
+Category: Releases
+Slug: smithay-v-0-1
+Authors: Victor Berger
+Summary: Announcement of version 0.1 of smithay, and a general presentation of the project and its goals.
+
+A few days ago we released version 0.1 of the [smithay crate](https://crates.io/crates/smithay),
+its first release on crates.io. On this occasion, let me make a slightly detailed presentation of this
+wayland compositor library.
+
+First of all, let me give some context of wayland, in order to show what problem smithay tries to solve.
+If you already know about this protocol, feel free to jump to next section.
+
+## Wayland background
+
+The [wayland protocol](https://wayland.freedesktop.org/) aims at replacing the X11 protocol as the standart
+communication mean between graphical client apps and the display server on linux. The rationale for this is
+that given how X11 is old, has accumulated a lot of debt over the years, and is full of security holes, it's
+better to just start over with a new protocol. The project has been going for a few years now and adoption
+is getting pretty good. Notably Gtk/Gnome and Qt/KDE are now almost fully compatible, and some linux
+distributions have started enabling it by default.
+
+In a few points, here are the main difference between Wayland and X11:
+
+- Wayland is much smaller and barebones. We could push the analogy to the point of saying that Wayland
+  is to X11 what Vulkan is to OpenGL. For example, a Wayland client is expected to fully handle the
+  drawing of its content, and just hand a buffer to the server via shared memory, while the X11 protocol
+  embeds a log of drawing primitives (though many recent X11 apps already do it that way anyway).
```

Comment:
"a log of"?

Reply:
ahah, that should be "many" actually I think (I was thinking "a lot of")